lobe/views/mos/mos.py

```diff
@@ -225,16 +225,12 @@ def mos_test(id, uuid):
             flash("Þú hefur ekki aðgang að þessari síðu", category='error')
             return redirect(url_for("mos", id=id))
     mos = Mos.query.get(id)
-    mos_list = MosInstance.query.filter(MosInstance.mos_id == id).all()
-    mos_list_to_use = []
-    for i in mos_list:
-        if (i.selected and i.path):
-            mos_list_to_use.append(i)
-    random.shuffle(mos_list_to_use)
+    mos_configurations = mos.getConfigurations()
+    mos_list = mos_configurations[(mos.num_participants - 1) % len(mos_configurations)]
```

Comment:
We'll definitely want to do this in a more sophisticated way once we have some more time, as this can easily introduce imbalance when people don't complete the test.